src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -167,6 +168,14 @@ public void addEntities() {
 	 * @return all the players in this field.
 	 */
 	public abstract ArrayList<PlayerFish> getPlayers();
+	
+	/**
+	 * @return
+	 * 		All the entities in this field.
+	 */
+	public List<Entity> getEntities() {
+		return new ArrayList<Entity>(entities);
```

Comment:
Do you require a list, or would the a ConcurrentLinkedQueue also suffice? If so, instead of creating a copy into an ArrayList, you can simply return entities, which results in better performance

<performance